src/jquery.interactive-world-map.js

```diff
@@ -1,34 +1,80 @@
-$(function(){
+;$(function( $, window, document, undefined) {
+
+    var pluginName = "interactiveWorldMap";
+    var marginCountryPopupForLeft;
+    var marginCountryPopupForTop;
+    var dataCountries = {};
+    var countriesFill = {};
+    var colorHighlighted;
+    var patternHover;
+    var colorHover;
+    var pattern2;
+    var pattern;
+    var margins;
+
+    var defaults = {
+        hover : "#FF0000",
+        highlighted: "#848484",
+        marginCountryPopupForTop: 100,
+        marginCountryPopupForLeft: 60
```

Comment:
Again, simple naming conventions to make sure everyone understands, i.e.: `hover` --> `hoverColor`, `highlighted` --> `highlightColor`

Further down in the code I won't be able to understand if `hover` is a value or a function...